Bootcamp.ioc

```diff
@@ -1,158 +1,154 @@
 #MicroXplorer Configuration settings - do not modify
```

Comment:
You don't actually need to use cubeMX. Although you could get some things to work by changing the settings a bit, for this boot camp, you should be modifying the tim.c file directly.

When you amend your PR, you should undo the changes to this file